meet-data/apf/1503/entries.csv

```diff
@@ -0,0 +1,57 @@
+Jessie Rush,31,F,F1,56.93,60,-90,90,-100,90,45,-50,-55,45,90,100,110,110,245,1,APF,SBD,Wraps
+Justen Frank,39,M,M-O,103.87,110,-287.5,287.5,307.5,307.5,-240,240,-250,240,237.5,250,-260,250,797.5,1,AAPF,SBD,Wraps
+Kat Colson,64,F,F1,79.15,82.5,,,,,,,,,,,,,,DQ,AAPF,SBD,Wraps
+Kelie Willson,37,F,F1,66.22,67.5,92.5,97.5,100,100,52.5,-57.5,57.5,57.5,97.5,102.5,110,110,267.5,2,APF,SBD,Wraps
+Lynn Jones,52,F,F1,70.31,75,70,77.5,-82.5,77.5,42.5,-50,-50,42.5,110,120,-125,120,240,3,AAPF,SBD,Wraps
+Matt Graber,31,M,M-O,88.45,90,-272.5,272.5,295,295,152.5,165,-182.5,165,100,242.5,-272.5,242.5,702.5,1,AAPF,SBD,Wraps
+Matt Thompson,35,M,M-O,45.36,100,175,185,-195,185,137.5,147.5,160,160,207.5,227.5,250,250,595,3,APF,SBD,Wraps
+Nate McLaughlin,23,M,M-O,132,140,265,285,300,300,175,185,-195,185,247.5,272.5,287.5,287.5,772.5,1,AAPF,SBD,Wraps
+Patrick Elkin,30,M,M-O,108.86,110,155,170,182.5,182.5,117.5,127.5,-137.5,127.5,192.5,202.5,-210,202.5,512.5,6,APF,SBD,Wraps
+Phillip Schmidt,25,M,M-O,122.24,125,185,-207.5,207.5,207.5,142.5,147.5,-155,147.5,185,200,220,220,575,3,APF,SBD,Wraps
+Robert Fuller,44,M,M-O,141.97,140+,240,255,265,265,150,160,167.5,167.5,267.5,277.5,287.5,287.5,720,1,AAPF,SBD,Wraps
+Rock Hayford,15,M,M-O,58.51,60,92.5,95,97.5,97.5,55,-60,62.5,62.5,127.5,-142.5,142.5,142.5,302.5,1,AAPF,SBD,Wraps
+Rodney Wasche,31,M,M-O,113.85,125,307.5,320,-327.5,320,-215,215,-227.5,215,307.5,-320,-320,307.5,842.5,1,APF,SBD,Wraps
+Rosemary Squillace,22,F,F1,66.68,67.5,105,112.5,125,125,72.5,82.5,-92.5,82.5,120,140,147.5,147.5,355,1,AAPF,SBD,Wraps
+Sara Hill,38,F,F1,99.56,90+,97.5,107.5,-130,107.5,57.5,70,-82.5,70,97.5,117.5,130,130,307.5,3,AAPF,SBD,Wraps
+Sean Clymer,26,M,M-O,45.36,100,-325,-325,-330,,-220,-220,,,272.5,,,272.5,,DQ,APF,SBD,Wraps
+Shari Lindsley,32,F,F1,138.57,90+,125,150,-175,150,60,75,85,85,100,137.5,-167.5,137.5,372.5,1,AAPF,SBD,Wraps
+Stacy Hayford,32,F,F1,58.29,60,82.5,-95,-100,82.5,42.5,50,-52.5,50,100,110,-117.5,110,242.5,2,AAPF,SBD,Wraps
+Tabor Martin,25,M,M-O,107.05,110,215,227.5,237.5,237.5,132.5,140,-147.5,140,262.5,280,287.5,287.5,665,2,APF,SBD,Wraps
+Tyson Lytle,16,M,M-O,137.67,140,-190,207.5,227.5,227.5,167.5,185,-195,185,205,232.5,252.5,252.5,665,2,APF,SBD,Wraps
+Ashleigh Hoff,28,F,F1,74.84,75,,,,,52.5,57.5,62.5,62.5,,,,,62.5,1,APF,B,Raw
+Brad Slining,40,M,M-O,142.2,140+,,,,,-287.5,287.5,300,300,,,,,300,1,APF,B,Raw
+Chad Liedtke,49,M,M-O,144.47,140+,,,,,142.5,152.5,157.5,157.5,,,,,157.5,2,APF,B,Raw
+Jessica Lindsley,35,F,F1,88.68,90,,,,,37.5,47.5,55,55,,,,,55,1,AAPF,B,Raw
+Luke Uribe,34,M,M-O,108.41,110,,,,,217.5,237.5,-250,237.5,,,,,237.5,1,APF,B,Raw
+Michael Kotnour,51,M,M-O,114.31,125,,,,,162.5,-170,-170,162.5,,,,,162.5,2,AAPF,B,Raw
+Rodney Wasche,31,M,M-O,113.85,125,,,,,-215,215,-227.5,215,,,,,215,1,APF,B,Raw
+Shari Lindsley,32,F,F1,138.57,90+,,,,,60,75,85,85,,,,,85,1,AAPF,B,Raw
+Shawn Frankl,37,M,M-O,98.43,100,,,,,217.5,237.5,-250,237.5,,,,,237.5,1,APF,B,Raw
```

Comment:
Shawn Frankl is a Multi-ply competitor... I'm really not sure how to count this meet.

Reply:
Guess I'll just leave this one for now.